test/compilable/crlf_dshell.d

```diff
@@ -0,0 +1,52 @@
+module test;
+import dshell;
+int runTest()
+{
+    // Test CRLF and mixed line ending handling in D lexer.
+    const src = buildPath(RESULTS_TEST_DIR, TEST_NAME ~ ".d");
+    
+    src.createFile(
+          "#!/usr/bin/env dmd -run\r\n"
```

Comment:
please make this a single `q{}` string.

Reply:
It's got to have `\r\n` line endings....I don't think `q{}` strings guarantee that

Reply:
That won't work because `\r` isn't a valid D token and I think the whole point of this test is to test for `\r` and friends.

Reply:
ew.